implementations/scripts/utils.py

```diff
@@ -0,0 +1,63 @@
+import datetime
+import json
+
+
+def str_to_dt_other(date):
```

Comment:
I think this is not needed, see above where you use this function, and the discussion on `datetime`.